app/components/sidebars/main/channels_list/channel_item/index.js

```diff
@@ -30,6 +31,7 @@ function makeMapStateToProps() {
 
         let displayName = channel.display_name;
         let isBot = false;
+        let isUserGuest = false;
```

Comment:
this is a bit confusing having the function name as `isGuest` and then passing a prop as `isGuest` maybe do an alias for the function `import {isGuest as isGuestUser}  from 'app/utils/users;`